qa/suites/deepsea/build-validation/ses_cephfs_test_mount/cluster/roles.yaml

```diff
@@ -0,0 +1,10 @@
+roles:
+  - [client.salt_master]
+  - [mon.a, mgr.x]
+  - [mon.b, mgr.y]
+  - [mon.c, mgr.z]
+  - [osd.0]
+  - [osd.1]
+  - [osd.2]
+  - [osd.3]
+  - [osd.4]
```

Comment:
tahle konfigurace se hojně opakuje - opravdu všechny ty testy nutně potřebují cluster o devíti nódách? Nebo se některé obejdou s méně nódami?

Taky pokud dva nebo více testů používají identický `roles.yaml`, chtělo by to ho přesunout jinam např. `qa/deepsea/build-validation-9nodes.yaml` a ty duplicitní soubory nahradit symlinky.

Reply:
predelano v commitu a7d74295f15fcf56142d3fda3375455a7980acb4